selftests/unit/test_utils_memory.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from avocado.utils import memory
+
+
+class UtilsMemoryTest(unittest.TestCase):
+
+    def test_numa_nodes_with_memory(self):
+        file_values = [u"0\n", u"1-3", u"0-1,12-14\n"]
+        expected_values = [[0], [1, 2, 3], [0, 1, 12, 13, 14]]
+        for idx, value in enumerate(file_values):
```

Comment:
nitpick: how about `for value, exp in zip(file_values, expected_values)`? ;-)

Reply:
yeah, changed accordingly and pushed.